src/Microsoft.AspNetCore.Blazor/Components/ProviderBase.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Blazor.Components
+{
+    /// <summary>
+    /// Non-generic base class for <see cref="Provider{T}"/>.
+    /// </summary>
```

Comment:
Might as well say that it's the best class for all providers.

Reply:
We make the best classes.

Reply:
As per other discussions, replaced with internal interface `ICascadingValueComponent`, so the XML doc is gone.